pp_calibrate.py

```diff
@@ -655,6 +667,21 @@ def calibrate(filenames, minstars, manfilter, manualcatalog,
 
     logging.info('Done! -----------------------------------------------------')
 
+    # set flag keyword to SUCCESS
+    for filename in filenames:
+        hdulist = fits.open(filename, mode='update', verify='silentfix',
+                            ignore_missing_end=True)
+        header = hdulist[0].header
+        try:
+            header.set('PP_CALIB', 'SUCCESS')
+        except:
```

Comment:
I don't think that there should be need to intercept errors, since `PP_CALIB` already exists in the header.

Reply:
... the same applies to the other functions...